guides/vcharly/README.md

```diff
@@ -0,0 +1,184 @@
+- [Downloading and First-Time Startup of CentOS](#downloadingcentos)
+- [Setting up CentOS](#settingupcentos)
+- [Installing Apachce](#installapache)
+- [Installing the Database](#installdb)
+- [Installing PHP](#installphp)
+- [Adding a User to the Database](#adduser)
+- [Installing Wordpress](#installwordpress)
+- [Configuring Wordpress](#configwordpress)
+- [Installing Wordpress from Web GUI](#wordpressgui)
+- [Troubleshooting](#trouble)
+
+## Prerequisities <a id="prereq"></a>
+1. Have pfSense Running
+2. Have Ubuntu DNS Running
+3. Have Windows 10 Running
+4. Have ISP Gateway Running
+
+## Summary <a id="summary"></a>
+The CentOS machine will be hosting a public e-commerce site created using Wordpress. When you are finished, the site will be accessible from the WAN network through the Windows 10 machine.
+
+<p align="center">
+  <img src="chart.jpg?raw=true" alt="Topology Image"/>
+</p>
+
+## Downloading and First-Time Startup of CentOS <a id="downloadingcentos"></a>
+1. First, download the CentOS ISO. It can be found [here](https://www.centos.org/download/)
+
+    We are going to be using the CentOS Minimal ISO and install everything we need.
+
+    Once the ISO is done downloading, proceed to create a new virtual machine inside VMWare
+or Virtual Box using the downloaded ISO. Use the recommended settings for the allocated ram, storage, and cpu cores.
+And when asked for network adapter, allow CentOS to access the Host-Only, DMZ adapter.
+
+    Boot the CentOS vm to proceed with the installation.
+
+2. Once booted, choose the `Install CentOS 7` option.
+
+    Now you will be greeted with a `Welcome to CentOS 7` screen.
+
+    Choose your prefered language and procced to the `Installation Summary` screen.
+
+    Before the installation can begin, you must choose the installation destination. Click on `Installation Destination` and choose the virtual 
+hard drive you created for the vm.
+
+    Click `Done` and `Begin Installation`.
+
+3. While CentOS installs, you must setup a `Root Password` and create a `New User`.
+
+    Once the install is done, you will be promted to `Reboot`.
+
+4. The reboot process shouldn't need any user input. But, once it has booted up, login with the new user you created while CentOS was installing.
+
+    If you successfully logged in, Congragulations! You have installed CentOS. If you couldn't login, try login in with `localhost login: root`
+and `Password: (The root password you set)`. Once you login with root, **BE SURE YOU CREATE A NEW USER**. If that didn't work, try reinstalling CentOS from the beginning.
+
+## Setting up CentOS <a id="settingupcentos"></a>
+
+1. Execute `ip a` and take note of your network adapter. It should be something along the lines of `enp0s3` for Virtual Box vms and `ens33` for 
+VMWare vms.
+
+2. Type `su root` to obtain root
+
+3. And then, `vi /etc/sysconfig/network-scripts/ifcfg-enp0s3`. (If you are using a VMWare vm, use `vi /etc/sysconfig/network-scripts/ifcfg-ens33`)
```

Comment:
Once Jon guide is done, do another pull request and have programs like `vi` linked to his how-to-guide on linux programs. Just a note for the future.